src/custom/state/orders/hooks.ts

```diff
@@ -72,6 +72,19 @@ type UpdateLastCheckedBlockCallback = (updateLastCheckedBlockParams: UpdateLastC
 
 type GetOrderByIdCallback = (id: OrderID) => SerializedOrder | undefined
 
+type OrderTypeKeys = 'pending' | 'expired' | 'fulfilled' | 'cancelled'
+function _concatOrdersState(state: OrdersState[ChainId], keys: OrderTypeKeys[]) {
+  if (!state) return []
+
+  const firstState = state[keys[0]] || {}
+  const restKeys = keys.slice(1)
+
+  return restKeys.reduce((acc, nextKey) => {
+    const nextState = Object.values(state[nextKey] || {})
+    return acc.concat(nextState)
+  }, Object.values(firstState))
+}
```

Comment:
Maybe im not understanding, but why not using the spread operator with the default state or `Object.assign`?  Feels too complicated